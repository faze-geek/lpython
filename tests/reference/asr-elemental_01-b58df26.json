{
    "basename": "asr-elemental_01-b58df26",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/elemental_01.py",
    "infile_hash": "551c075e5247becad7cbb2517969ee76aac1ebade47b409e4c8fedde",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-elemental_01-b58df26.stderr",
    "stderr_hash": "02d017beacd5580dd338ee82f4a3c567a1a1dc2c604f1b8f4347bb4e",
    "returncode": 2
}