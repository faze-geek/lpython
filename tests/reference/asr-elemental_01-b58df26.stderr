semantic error: 'f64[101]' object does not support item assignment
  --> tests/../integration_tests/elemental_01.py:63:9
   |
63 |         array_a[i] = float(i)
   |         ^^^^^^^^^^^^^^^^^^^^^ 
